scripts/HNG-01544.py

```diff
@@ -1,2 +1,2 @@
 
-print ("Hello world,this is Adeniyi Samuel with HNGi7-01544 using Python For stage 2 task. [sa7259379@gmail.com] ")
+print ("Hello world,this is Adeniyi Samuel with HNGi7-01544 using Python For stage 2 task. sa7259379@gmail.com ")
```

Comment:
Your string format isn't correct and you will have issues when I merger it.
Please, use the correct string format from the group HNGi7 ID